app/services/bulk_update/recommendations_uploads/create_recommendations_upload_rows.rb

```diff
@@ -36,11 +36,11 @@ def call
 
       attr_reader :recommendations_upload, :csv
 
-      def create_recommendations_upload_row!(trainee, row, row_number)
+      def create_recommendations_upload_row!(trainee, row, row_number, date)
         recommendations_upload.rows.create(
           matched_trainee_id: trainee&.id,
           csv_row_number: row_number,
-          standards_met_at: row.standards_met_at&.to_date,
+          standards_met_at: date,
```

Comment:
I'm not 100% sure about this column being the only one where we're not saving what the user actually entered.

At the moment the error CSV will still work because we're generating it from the stored CSV (which will have the bad date). But if we change it so that we need to re-create it from DB entries then we won't be able to reproduce what they've entered, so maybe we should wait for a decision on that.

If we move towards saving a json representation or something of their whole row, then this might be ok.